ui/components/app/flask/snap-settings-card/snap-settings-card.js

```diff
@@ -0,0 +1,278 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+import Card from '../../../ui/card';
+import Box from '../../../ui/box';
+import IconWithFallback from '../../../ui/icon-with-fallback';
+import IconBorder from '../../../ui/icon-border';
+import Typography from '../../../ui/typography/typography';
+import ToggleButton from '../../../ui/toggle-button';
+import Chip from '../../../ui/chip';
+import ColorIndicator from '../../../ui/color-indicator';
+import Button from '../../../ui/button';
+
+import {
+  COLORS,
+  TYPOGRAPHY,
+  FONT_WEIGHT,
+  ALIGN_ITEMS,
+  DISPLAY,
+  TEXT_ALIGN,
+} from '../../../../helpers/constants/design-system';
+
+const STATUSES = {
```

Comment:
we could pull the enum type exported from snap controller: https://github.com/MetaMask/snaps-skunkworks/blob/main/packages/controllers/src/snaps/SnapController.ts#L156-L161

or pull that out into a more common `@metamask/snap-types` package

Reply:
but that's typescript though no? how would we use that enum with regular old react?